custom_components/dyson_local/manifest.json

```
@@ -7,5 +7,6 @@
     "documentation": "https://github.com/libdyson-wg/ha-dyson",
     "iot_class": "local_push",
     "issue_tracker": "https://github.com/libdyson-wg/ha-dyson/issues",
-    "version": "1.3.10"
+    "version": "1.3.10",
+    "import_executor": "true"
```

```suggestion
    "import_executor": true
```

This changes the type of this value from a string to a boolean, and should help the automated test pass.